package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "node-notifier",
-  "version": "6.0.0",
+  "version": "7.0.0",
```

@yoavain - We'll version bump in a separate PR, but what was the reasoning you were thinking for bumping this to v7?
```suggestion
  "version": "6.0.0",
```

---

If you look only at the SemVer definition, this should be a minor update, as it is backward compatible and only adds functionality. However, there are a couple of reasons to set this as a major update. For starters, I updated SnoreToast binaries, so this can cause issues on some environments (For example, antivirus false positives).
Also, I understand this package is being downloaded by millions every week, so I think it's best not to push such update to so many people automatically (assuming big part of them auto update minor/patch). So, if we "force" a manual update, if there is a bug, it will be spreaded to a smaller portion of your clients.
But it's your call...

---

Thanks for explaining :) I generally agree with your reasoning, particularly the point about changing the SnoreToast binaries. Regardless we'll do the actual version bump in a different PR (since we usually try to bundle changes together for a release), so if you could apply the suggestion/remove the version diff that would be awesome :)

---

Done